app/[locale]/apps/[application]/page.tsx

```diff
@@ -254,14 +255,16 @@ const Page = async ({
                           <p className="text-primary group-hover:text-primary-hover">
                             {nextApp.name}
                           </p>
-                          <LinkOverlay
-                            href={`/apps/${slugify(nextApp.name)}`}
-                            matomoEvent={{
-                              eventCategory: "detail",
-                              eventAction: `app name ${app.name}`,
-                              eventName: "see_next",
-                            }}
-                          />
+                          <LinkOverlay asChild>
```

Comment:
Correct me if I'm wrong, but from what I can tell this pattern applied to every usage of `LinkOverlay` in the codebase... we should update that component to automatically handle the `asChild` and `BaseLink` embedding